bedrock/firefox/firefox_details.py

```diff
@@ -246,13 +249,13 @@ def get_download_url(self, channel, version, platform, locale,
                 suffix = 'latest'
 
             _version = ('beta-' if channel == 'beta' else '') + suffix
-        elif not funnelcake_id:
+        elif not _include_funnelcake_param:
             # Force download via SSL. Stub installers are always downloaded via SSL.
             # Funnelcakes may not be ready for SSL download
             _version += '-SSL'
 
         # append funnelcake id to version if we have one
-        if funnelcake_id:
+        if _include_funnelcake_param:
```

Comment:
Note: I'm only changing the logic for links where `force_direct` is `True`. I could also potentially apply this logic for transitional links below (e.g. `/firefox/new/?scene=2`), so it would only be `/firefox/new/?scene=2&f=99` for Windows. Thoughts?

I decided against doing this initially since we're running an experiment that shows a new design for the download page based on funnelcake params. If someone visits `/firefox/new/?f=99` on macOS for example, they will still link to `/firefox/new/?scene=2&f=99` when clicking the button. This means they will get the correct corresponding templates still, only their download now won't break.

If we think this is an edge case and makes things more confusing than it's worth, I'd be happy to update and only include the `f` param for Windows / en-US on transitional links too.